meteor/lib/tv2/AHKkeyboardMap.ts

```diff
@@ -0,0 +1,75 @@
+import * as _ from 'underscore'
```

Comment:
Should this file even be included in the repo?

Reply:
We'll allow it for now. Ref: https://github.com/nrkno/sofie-core/pull/694#discussion_r841429702